cmd/ztest/ztest.c

```diff
@@ -7384,7 +7384,7 @@ main(int argc, char **argv)
 	 * segments can be reconstructed in a reasonable amount of time
 	 * when reconstruction is known to be possible.
 	 */
-	zfs_reconstruct_indirect_damage_fraction = 4;
+	zfs_reconstruct_indirect_damage_fraction = 100;
```

Comment:
Do we want to maybe at to that comment the points discussed with Matt and Brian?
For example, something along those lines?
```
/*
 * Verify that even extensively damaged split blocks with many
 * ...
 * when reconstruction is known to be possible.
 *
 * Note: Keep in mind that the lower this value is, the more damage we inflict,
 * and the more time ztest spends in recovering that damage. We chose the
 * default of inducing damage 1/100th of the time so recovery is tested enough
 * but not too frequently to do so in the expense of other types of testing.
 */
```